vignettes/a07_updating_rsyncrosim_scripts_to_v2.Rmd

```diff
@@ -0,0 +1,117 @@
+
+*Note*: <a href="https://syncrosim.com/studio-download/" target="_blank">SyncroSim v3.0.9</a> or higher is required to use `rsyncrosim v2.0.0` or higher. 
+
+## Updated functions
+
+### Installing/uninstalling packages
+
+Previously, the `addPackage` and `removePackage` functions were used to install and uninstall packages from SyncroSim. Now to install and uninstall packages from SyncroSim, use:
+
+```{r install packages, eval=FALSE}
+installPackage()   # Installs the selected package(s) to SyncroSim
+uninstallPackage() # Uninstalls the selected package(s) from SyncroSim
+```
+
+### Adding/removing packages
+
+Now you can use multiple SyncroSim packages in a single library. To add and remove packages from your library, use:
+
+```{r add package, eval=FALSE}
+addPackage()    # Adds package(s) to the library
+removePackage() # Removes package(s) from the library.
+```
+Note that when you remove a package from your library any associated datasheets will be removed as well.
+
+## Deprecated functions
+
+In SyncroSim 3, the concept of addon packages no longer exists. All addon packages have either been converted to standalone packages (e.g., `burnP3PlusPrometheus`) and can be added to a library without having to load a base package first, or incorporated directly into the base packages (e.g., `stsimsf` is now part of `stsim`). Therefore, the following functions have been removed:
+
+```{r deprecated, eval=FALSE}
+addon()
+disableAddon()
+enableAddon()
+```
+
+## Library
+
+`info(ssimLibrary)` no longer shows the *Package name*, *Current package version* and *Minimum package version*, and the Input and Output folder paths have been combined into the *Data folder*.
+
+The `package` argument in the `ssimLibrary(package)` function has been renamed to `packages`
+
+```{r library, eval=FALSE}
+myLibrary <- ssimLibrary(packages = c())
+```
+
+The `addon` and `template` arguments have also been removed.
+
+## Dependencies
+
+To view the existing dependencies for a scenario, use:
+
+```{r view dependencies, eval=FALSE}
+dependency(myScenario)
+```
+
+To set dependencies for a scenario, use:
+
+```{r set dependencies, eval=FALSE}
+dependency(myScenario) <- c("Scenario 1", "Scenario 2")
+```
+
+Note that this will remove any previously set dependencies.
+Also, the order in which the dependencies are listed can be important. In this example, `Scenario 1` takes precedence over `Scenario 2`.
+
+
+## Multiprocessing
+
+The `run()` function no longer has the `jobs` argument. Use the `core_Multiprocessing` datasheet to set the number of cores to use. Note that this will affect every scenario run.
+
+
+## Variable naming
+
+The **ID** in variable names have been changed to **Id**. It is generally safe to substitute `ID` for `Id` throughout your script, unless that affects any variables not related to SyncroSim.
```

Comment:
I think this also needs more detail. It's specifically the "ID" in the primary key that has been updated. Some datasheets might have additional columns that have "ID" in them that the package developer has set. Give example